docker/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# set -o xtrace
```

Comment:
Yes, exactly. I used it during development and I thought it'd be nice to leave it there in case it's needed again.